### DISCUSSION TITLE
FileUpload pt badge dosn't work

### DISCUSSION THREAD

**Discussion by HaukeOtt** [primefaces/community#375] [PrimeVue] [answered]
Hi, I'm new to PrimeVue and dont know why this isn't working.
I'm trying to remove this badge:
<img width="49" alt="image" src="https://github.com/primefaces/community/assets/118479690/4f8f0c51-fe0e-4c74-b7dd-bd495d1241ee">
I tryed this code:
```vue
<FileUpload customUpload @uploader="uploadHandler" chooseLabel="Wähle" uploadLabel="Hochladen"
      :showCancelButton="false" :pt="{
      badge:{style:'display: none !important;'}
      }"/>
```
But the Browser dosen't show any differences and I have no error thrown:
```html
<span class="p-badge p-component p-badge-warning p-fileupload-file-badge" data-pc-name="badge" data-pc-section="root">Pending</span>
```
thanks in advance

**Comment by FlipWarthog** [comment]
Not sure if it's an issue that needs to be addressed, but it looks like you need to reference the badge's root:
```js
:pt="{
  badge: { root: { style: 'display: none !important;' } },
}"
```

**Comment by tugcekucukoglu** [comment, marked as answer]
Hi,

FileUpload uses Badge component internally. So the implementation should be like this:

```
<FileUpload customUpload @uploader="uploadHandler" chooseLabel="Wähle" uploadLabel="Hochladen"
      :showCancelButton="false" :pt="{
      badge:{
         root: {
            style:'display: none !important;'
         }
       }
}"/>

```

**Reply by nikola-djokovic-web** [reply]
Hi. How to edit the text inside the badge? I want to translate it based on app localization.
